pkgs/development/libraries/wt/cmake.patch

```diff
@@ -0,0 +1,10 @@
+--- a/CMakeLists.txt	2016-07-13 14:27:26.000000000 +0200
```

Comment:
I wrote this patch to fix a cmake warning but it's not essential.
I sent a PR to upstream.